pkg/controllers/provisioning/v1alpha1/allocation/controller.go

```diff
@@ -46,13 +45,13 @@ type Controller struct {
 }
 
 // For returns the resource this controller is for.
-func (c *Controller) For() controllers.Object {
```

Comment:
did we delete the definition of controllers.Object now that it's not used?

Reply:
Yes.